### PR TITLE
Bump base and atomistics together

### PR DESCRIPTION
**Pull request by pmrv**

### CODE REVIEW

File: .binder/environment.yml [hunk 1]
```diff
@@ -7,8 +7,8 @@ dependencies:
 - codacy-coverage
 - matplotlib =3.8.2
 - numpy =1.26.2
-- pyiron_base =0.6.14
-- pyiron_atomistics =0.4.1
+- pyiron_base =0.6.19
```
Comment (jan-janssen) [inline .binder/environment.yml:10]:
```suggestion
- pyiron_base =0.6.20
```

File: .ci_support/environment.yml [hunk 2]
```diff
@@ -7,8 +7,8 @@ dependencies:
 - codacy-coverage
 - matplotlib =3.8.2
 - numpy =1.26.2
-- pyiron_base =0.6.14
-- pyiron_atomistics =0.4.1
+- pyiron_base =0.6.19
```
Comment (jan-janssen) [inline .ci_support/environment.yml:10]:
```suggestion
- pyiron_base =0.6.20
```

File: docs/environment.yml [hunk 3]
```diff
@@ -9,8 +9,8 @@ dependencies:
 - codacy-coverage
 - matplotlib =3.8.2
 - numpy =1.26.2
-- pyiron_base =0.6.14
-- pyiron_atomistics =0.4.1
+- pyiron_base =0.6.19
```
Comment (jan-janssen) [inline docs/environment.yml:12]:
```suggestion
- pyiron_base =0.6.20
```

File: setup.py [hunk 4]
```diff
@@ -32,15 +32,15 @@
     install_requires=[        
         'matplotlib==3.8.2',
         'numpy==1.26.2',
-        'pyiron_base==0.6.14',
+        'pyiron_base==0.6.19',
```
Comment (jan-janssen) [inline setup.py:35]:
```suggestion
        'pyiron_base==0.6.20',
```

**Review by jan-janssen — COMMENTED** [review]
Is it possible to update `pyiron_base` to 0.6.20 ?

**Comment by pmrv** [comment]
> Is it possible to update `pyiron_base` to 0.6.20 ?

I had left it at .19 because the atomistics version 0.4.4 still depends on that and hasn't been bumped its base dependency.  I only want this PR now, so that #949 is in a shape where @liamhuber can use it immediately.

**Comment by jan-janssen** [comment]
> > Is it possible to update `pyiron_base` to 0.6.20 ?
> 
> I had left it at .19 because the atomistics version 0.4.4 still depends on that and hasn't been bumped its base dependency. I only want this PR now, so that #949 is in a shape where @liamhuber can use it immediately.

No - 0.4.4 supports 0.6.20 - https://github.com/conda-forge/pyiron_atomistics-feedstock/blob/main/recipe/meta.yaml#L54

**Comment by pmrv** [comment]
Strange, because this is not shown in the changelog of atomistics nor was there a PR for it.

**Comment by jan-janssen** [comment]
> Strange, because this is not shown in the changelog of atomistics nor was there a PR for it.

The changes were included in https://github.com/pyiron/pyiron_atomistics/pull/1283 and released to conda-forge as part of https://github.com/conda-forge/pyiron_atomistics-feedstock/pull/134

**Comment by pmrv** [comment]
Notebook failure is related to tinybase notebooks that are fixed in #949, so I'll merge this anyway.

**Pull request merged**